CHANGELOG.md

```diff
@@ -8,6 +8,34 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Added
 
+### Changed
+
+### Fixed
+
+### Documentation
+
+## [2.8.0]
```

Comment:
I think that the changes for 2.8.0 should be added separately, after the 2.7.0 is released. 

It could be confusing to release version 2.7.0 with CHANGELOG for 2.8.0 :wink:

Reply:
2.8.0 has been tagged so the change list is complete there; merging & releasing 2.8.0